docs/troubleshoot/app-framework/app-troubleshoot.md

```diff
@@ -1,8 +1,194 @@
+
+    ```
+    ZIS status - Failure (name='ZWESIS_STD      ', cmsRC=33, description='Permission denied'
+    ```
+    Check that the user ID of the ZWESVSTC started task is authorized to access the load module. Only authorized code can call ZWESIS01 because it is an APF-authorized load module. The setup for each security manager is different and is documented in the section "Security requirements for the cross memory server" in the topic  [Manually installing the Zowe Cross Memory Server](../../user-guide/install-zos.html#manually-installing-the-zowe-cross-memory-server).
+
+    **Note** If you are using RACF security manager, a common reason for seeing `Permission Denied` is that the user running the started task ZWESVSTC (typically ZWESVUSR) does not have READ access to the FACILITY class ZWES.IS.
+
+    If the message includes the following text, the configuration of the Application Framework server may be incomplete:
+
+    ```
+    ZIS status - Failure read failed ret code 1121 reason 0x76650446
+    ```
+    If you are using AT/TLS, then the ```"attls" : true``` statement might be missing from the ```zluxserver.json``` file. For more information, see [Configuring Zowe App Server for HTTPS communication with ZSS](../../user-guide/mvd-configuration.html#configuring-zss-for-https)
+
+
+### ZLUX unable to communicate with zssServer
+
+On the [Zowe architecture diagram](../../getting-started/zowe-architecture.md) there is a communication between ZLUX which is a Node.js runtime servering the Zowe desktop web pages to the user's browser, and the zssServer running on port 8542.  If this communication is failing then the user will be unable to log onto the desktop.
+
+Follow these steps: 
+1. Open the log file `$INSTANCE_DIR/logs/appServer-yyyy-mm-dd-hh-ss.log`.  This file is created each time ZWESVSTC is started and only the last five files are kept.  
+
+2. Look for the message that starts with `GetAddrInfoReqWrap.onlookup` and the log messages below.  
+
+```
+yyyy-mm-dd hh:mm:ss.ms <ZWED:16842977> ZWESVUSR INFO (_zsf.apiml,apiml.
+yyyy-mm-dd hh:mm:ss.ms <ZWED:16842977> ZWESVUSR INFO (_zsf.auth,webauth
+yyyy-mm-dd hh:mm:ss.ms <ZWED:16842977> ZWESVUSR WARN (_zsf.proxy,proxy.
+  at GetAddrInfoReqWrap.onlookup Ýas oncomplete¨ (dns.js:64:26) {    
+  errno: 'ENOTFOUND',                                                  
+  code: 'ENOTFOUND',                                                    
+  syscall: 'getaddrinfo',                                              
+  hostname: 'localhost'                                                
+}      
+``` 
+3. Map localhost to port 127.0.0.1
+
+Create an entry in the file `/etc/hosts` that contains the line
+
+```
+localhost        127.0.0.1
+```
+
+4. Restart the `ZWESVSTC` address space.
+
+## Server startup problem ret=1115
+
+**Symptom:**
+When ZWESVSTC is restarted, the following message is returned in the output of the ZSS server log file, `$INSTANCE_DIR/logs/zssServer-yyyy-mm-dd-hh-ss.log`:

```

Comment:
"ZSS server" is more readable and "zssServer" is the name of the executable, but you used the latter before ...